tests/verifications/REPORT.md

```diff
@@ -15,12 +15,62 @@
 
 | Provider | Pass Rate | Tests Passed | Total Tests |
 | --- | --- | --- | --- |
+| Llama_meta_ref | 100.0% | 26 | 26 |
 | Together | 51.3% | 39 | 76 |
 | Fireworks | 47.4% | 36 | 76 |
 | Openai | 100.0% | 52 | 52 |
 
 
 
+## Llama_meta_ref
```

Comment:
nit: `llama_meta_ref` is kinda opaque as a key. why is `llama` there in this name? could we name it `Meta_Reference`?